apps/modernization-ui/src/styles/_uswds.scss

```diff
@@ -56,6 +56,7 @@ table.usa-table {
 .usa-modal__main {
     margin: 0 !important;
     padding: 0 !important;
+    overflow: auto;
```

Comment:
Suggestion: Applying `overflow: auto` here will affect every modal in the application. This may or may not be a problem but should be done with the appropriate level of caution.

It may be safer to only apply the change to the specific modal instance in which it is needed.